ci/check.sh

```diff
@@ -44,10 +44,9 @@ if [[ "$1" == "--check" ]]; then
 fi
 
 banner "Executing in conda environment ${CONDA_DEFAULT_ENV} in directory ${root}"
+run "Style Checking" "ruff format fgpyo"
+run "Linting"        "ruff check --fix fgpyo"
```

Comment:
I'm loosing track of the various PRs ... but will we want to include `tests` in this too?

Reply:
Yes, but it will have to wait until `tests/` gets refactored into a top level directory, which I think is happening in #135